packages/core/src/components/Menu/MenuItem/__stories__/MenuItem.stories.tsx

```diff
@@ -18,60 +21,68 @@ export default {
   component: MenuItem,
   argTypes: metaSettings.argTypes,
   decorators: metaSettings.decorators
-};
+} satisfies Meta<typeof MenuItem>;
 
 const menuItemTemplate = (args: MenuItemProps) => (
   <Menu>
     <MenuItem {...args} />
   </Menu>
 );
 
-export const Overview = {
+export const Overview: Story = {
   render: menuItemTemplate.bind({}),
-  name: "Overview",
-
   args: {
     title: "Menu item"
+  },
+  parameters: {
+    docs: {
+      liveEdit: {
+        isEnabled: false
+      }
+    }
   }
 };
 
-export const States = {
+export const States: Story = {
   render: () => (
     <Menu>
       <MenuItem title="Regular menu item" />
       <MenuItem title="Selected menu item" selected />
       <MenuItem title="Disabled menu item" disabled />
     </Menu>
-  ),
-  name: "States"
+  )
 };
 
-export const Icons = {
+export const Icons: Story = {
   render: () => (
     <Menu>
       <MenuItem title="SVG icon" icon={Activity} />
       <MenuItem title="Font icon" icon="fa fa-star" iconType={MenuItem.iconType.ICON_FONT} />
     </Menu>
   ),
-  name: "Icons"
+  parameters: {
+    docs: {
+      liveEdit: {
+        scope: { Activity }
```

Comment:
What it this?

Reply:
```
<MenuItem title="SVG icon" icon={Activity} />
```

I need to add this icon to the scope
otherwise it will only know what is VibeIcons.Activity, but the story says Activity